issueTemplate.md

```diff
@@ -0,0 +1,17 @@
+**WARNING: Issue will be closed if it does not follow these guidelines.
```

Comment:
Should this line be commented out, as the other instruction are?

Reply:
This does not necessarily need to be commented out unless you feel that it should be. I was basing this template off of another repository which had a similar 'warning' message regarding not following the guidelines. But it does not matter to me either way!